javascript/javascript2/week3/The fastest presser in this realm.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+// counter for L
+let totalL = 0;
+const elL = document.querySelector("#countL");
+const pressedL = (e) => {
+  if (e.keyCode == 76) {
+    totalL++;
+    elL.innerHTML = totalL;
+  }
+};
+function counterL() {
+  document.addEventListener("keydown", pressedL);
+}
+// counter for S
+
+let totalS = 0;
+const elS = document.querySelector("#countS");
+const pressedS = (e) => {
+  if (e.keyCode == 83) {
+    totalS++;
+    elS.innerHTML = totalS;
+  }
+};
+function counterS() {
+  document.addEventListener("keydown", pressedS);
+}
+
+// setting the time of the game
+
+// const time = timeSet.value;
+const startButton = document
```

Comment:
This is a bit tricky but startbutton is the result of what `.addEventListener` returns! That might be the element but it also might not